src/include/hip/rpp_hip_common.hpp

```diff
@@ -497,13 +514,28 @@ __device__ __forceinline__ void rpp_hip_load8_and_unpack_to_float8(uchar *srcPtr
     src_f8->y = rpp_hip_unpack(src.y);
 }
 
+__device__ __forceinline__ void rpp_hip_load8_and_unpack_to_float8_mirror(uchar *srcPtr, int srcIdx, d_float8 *src_f8)
```

Comment:
probably you can pass (srcPtr + srcIdx) directly to this function (remove srcIdx), applies to all such calls

Reply:
Done